config/urls.py

```diff
@@ -1,8 +1,14 @@
-"""Project URL Configuration"""
+from django.conf.urls import url
 from django.contrib import admin
+from rest_framework import routers
 from django.urls import path, include
 
+from quran.api import views as quran_views
+
+router = routers.SimpleRouter()
+router.register(r'quran/surah', quran_views.SurahListRetrieveView, base_name="quran")
+
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', include('quran.urls')),
+    url(r'^api/', include((router.urls, 'api'), namespace='api')),
```

Comment:
I don't think we need regexes here: https://docs.djangoproject.com/en/2.1/topics/http/urls/#using-regular-expressions

Reply:
oooh, yeap.